src/manifests/manifest.py

```diff
@@ -12,7 +12,7 @@
 
 class Manifest(ABC):
     SCHEMA = {
-        "schema-version": {"required": True, "type": "string", "allowed": ["1.0"]}
+        "schema-version": {"required": True, "type": "string", "allowed": [""]}
```

Comment:
This says that only a blank string value is allowed, and the corresponding test was updated as such. This isn't the case. Remove the `allowed` field altogether, which would allow anything, and write a test that checks that you can put 1.0, and that you cannot put no value.